src/Slim/Middleware/Session.php

```diff
@@ -43,6 +43,7 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'csrf'        => false
```

Comment:
I put `'csrf'` key inside `$settings` array because not all pages should be protected by CSRF, so it's possible to enable/disable at will inside the same project. 

If I only check the presence of `slim/csrf`, then maybe we can run `startSession` inside the constructor without any controls, starting the session directly in the middleware.

What's your opinion?